d2l-icon.html

```diff
@@ -0,0 +1,75 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-icon/iron-icon.html">
+<link rel="import" href="../d2l-colors/d2l-colors.html">
+<link rel="import" href="../d2l-polymer-behaviors/d2l-dom.html">
+<dom-module id="d2l-icon">
+	<template>
+		<style include="d2l-colors">
+			:host {
+				color: var(--d2l-color-tungsten);
+				--iron-icon-height: 18px;
+				--iron-icon-width: 18px;
+			}
+			:host([icon*="d2l-tier2:"]) {
+				--iron-icon-height: 24px;
+				--iron-icon-width: 24px;
+			}
+			:host([icon*="d2l-tier3:"]) {
+				--iron-icon-height: 30px;
+				--iron-icon-width: 30px;
+			}
+			:host-context([dir="rtl"]) iron-icon {
+				transform: scaleX(-1);
+			}
+			:host([no-rtl]) iron-icon {
+				transform: none;
+			}
+		</style>
+		<iron-icon icon="[[icon]]"></iron-icon>
+	</template>
+	<script>
+		Polymer({
+			is: 'd2l-icon',
+			properties: {
+				icon: String,
+				noRtl: {
+					type: Boolean,
+					readOnly: true,
+					reflectToAttribute: true,
+					value: false
+				}
+			},
+			ready: function() {
+				var isNoRtl = this.__isNoRtl();
+				this._setNoRtl(isNoRtl);
+			},
```

Comment:
If I understand this correctly, the need for looking at the composed DOM is because the `svg` is contained within `iron-icon`'s shadow-DOM, and so a standard querySelector won't find it?